adapters/postgres/postgres.go

```diff
@@ -133,6 +132,67 @@ func WhereByRequest(r *http.Request, initialPlaceholderID int) (whereSyntax stri
 	return
 }
 
+// SetByRequest create a set clause for SQL
+func SetByRequest(r *http.Request, initialPlaceholderID int) (setSyntax string, values []interface{}, err error) {
+	body := make(map[string]interface{})
+	if err = json.NewDecoder(r.Body).Decode(&body); err != nil {
+		return
+	}
+	defer r.Body.Close()
+	fields := make([]string, 0)
+	for key, value := range body {
+		if chkInvalidIdentifier(key) {
+			err = errors.New("Set: Invalid identifier")
+			return
+		}
+		fields = append(fields, fmt.Sprintf("%s=$%d", key, initialPlaceholderID))
+
+		switch value.(type) {
+		case []interface{}:
+			values = append(values, parseArray(value))
+		default:
+			values = append(values, value)
+		}
+
+		initialPlaceholderID++
+	}
+	setSyntax = strings.Join(fields, ", ")
+	return
+}
+
+// InsertByRequest create insert SQL
+func InsertByRequest(r *http.Request) (colsName string, colsValue string, values []interface{}, err error) {
```

Comment:
@marioidival no, we will break the compatibility in next release. this PR is first step to turn pREST more easy to  as a package